### ISSUE TITLE
ibm-table AVT issue

### ISSUE THREAD

**Issue by aishwaryamyana** [IBM/carbon-components-angular#1617]
ibm-table accessibility violation issue - Complex table does not have headers for each cell properly defined with 'header' or 'scope'

<img width="1680" alt="Screen Shot 2020-11-18 at 1 37 54 PM" src="https://user-images.githubusercontent.com/53562295/99502660-6670ab00-29a3-11eb-953f-7826480e3f70.png">
